src/devices/digital_input.rs

```diff
@@ -11,10 +11,7 @@
 //
 
 use crate::{AttachCallback, DetachCallback, GenericPhidget, Phidget, Result, ReturnCode};
-use phidget_sys::{
-    self as ffi, PhidgetDigitalInputHandle as DigitalInputHandle,
-    PhidgetDigitalOutputHandle as DigitalOutputHandle, PhidgetHandle,
-};
+use phidget_sys::{self as ffi, PhidgetDigitalInputHandle, PhidgetHandle};
```

Comment:
Why go with the redundantly long name? Just curious.

Reply:
I had removed the as in my original pass through. I did this PR after recloning your work to make prepare the different PRs, I might have forgotten it. I'd rather keep the original PhidgetDigitalOutputHandle  as it is easier to work with vscode's IDE. well correct this